GwApiService/src/main/java/us/gameandwatching/gwapi/service/calc/EloCalculator.java

```diff
@@ -0,0 +1,81 @@
+package us.gameandwatching.gwapi.service.calc;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import us.gameandwatching.gwapi.core.EloPair;
+import us.gameandwatching.gwapi.core.MatchResult;
+
+@Singleton
+public class EloCalculator {
+
+  @Inject
+  public EloCalculator() {
```

Comment:
If we never actually inject anything and this is a `@Singleton` class, then I think that means it could just be a class with some static methods that is never actually instantiated?